src/applications/disability-benefits/2346/Components/ConfirmAddress.jsx

```diff
@@ -0,0 +1,73 @@
+import PropTypes from 'prop-types';
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+import { getVeteranAddressInfo } from '../actions';
+
+class ConfirmAddress extends Component {
+  static propTypes = {
+    addressLine1: PropTypes.string.isRequired,
+    addressLine2: PropTypes.string.isRequired,
+    city: PropTypes.string.isRequired,
+    state: PropTypes.string.isRequired,
+    country: PropTypes.string.isRequired,
+    zip: PropTypes.string.isRequired,
+    email: PropTypes.string.isRequired,
+    getVeteranAddressInfo: PropTypes.func.isRequired,
+  };
+  static defaultProps = {
+    addressLine1: '',
+    addressLine2: '',
+    city: '',
+    state: '',
+    country: '',
+    zip: '',
+    email: '',
+    getVeteranAddressInfo: () => {},
+  };
+  componentDidMount() {
+    this.props.getVeteranAddressInfo();
+  }
+
+  render() {
+    const { addressLine1, addressLine2, city, state, zip, email } = this.props;
+
+    return (
+      <div>
+        <h1>Confirm your address</h1>
+        <div>
+          <label htmlFor="address1">Address Line 1</label>
+          <input id="address1" type="text" defaultValue={addressLine1} />
+          <label htmlFor="address2">Address Line 2</label>
+          <input type="text" defaultValue={addressLine2} />
+          <label htmlFor="city">City</label>
+          <input type="text" id="city" defaultValue={city} />
+          <label htmlFor="state">State</label>
+          <input type="text" id="state" defaultValue={state} />
+          <label htmlFor="zip">Zip</label>
```

Comment:
Please change this to "Postal code" instead of "Zip". Users outside of the U.S. may not know what zip code means.

Reply:
Have you thought about using the review cards to show this info?

![Screen Shot 2020-02-12 at 1 31 06 PM](https://user-images.githubusercontent.com/136959/74370004-fee65900-4d9b-11ea-9a07-e0026295e6df.png)

I haven't seen the design for your page, but this is a nice solution.

Reply:
@Mottie Where is the element located so we send it over to the design team for their approval.

Reply:
The postal code update has been made

Reply:
See https://github.com/department-of-veterans-affairs/vets-website/blob/master/src/applications/disability-benefits/996/pages/contactInformation.js - I know Riley is familiar with it, she's the one that helped design the HLR form!

Reply:
So @Mottie We are not going to go with the review cards right now per our designers here is the slack thread https://dsva.slack.com/archives/CPE4AJ6Q0/p1581540970390300?thread_ts=1581539599.389500&cid=CPE4AJ6Q0